go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -27,6 +26,10 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
+
```

Comment:
nit: this blank line is unnecessary, can you remove it?

Reply:
fixed